tools/internal/ocpaths/clone_public.go

```diff
@@ -18,6 +19,10 @@ func ClonePublicRepo(downloadPath, branch string) (string, error) {
 	}
 	publicPath := filepath.Join(downloadPath, "public")
 
+	if _, err := os.Stat(publicPath); err == nil { // If NO error
```

Comment:
could this cause detection of an old/stale repo?

Reply:
Yes, the purpose is to avoid having to download every single time. Let me add more documentation on this, thanks for catching.